src/main/java/com/google/sps/servlets/GetUsersServlet.java

```diff
@@ -75,23 +75,23 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
 			// Get display name, email and lastActive
 			String name = user.getString("name");
-			String email = user.getString("email");
 			String lastActive = user.getString("lastActive");
 
 			// Determine if user is active
 			boolean isActive = userIsActive(lastActive);
-			users.add(new FormattedUser(partner, name, email, isActive, true));
+			users.add(new FormattedUser(partner, name, isActive, true));
 		}
 
 		for (String appUser : nonCollaborators) {
-			// Pull user from datastore
-			Key key = datastore.newKeyFactory().setKind("User").newKey(appUser);
-			Entity user = datastore.get(key);
-
-			// Get display name, email and lastActive
-			String name = user.getString("name");
-			String email = user.getString("email");
-			users.add(new FormattedUser(appUser, name, email, true, false));
+			if (!appUser.equals(username)) {
+				// Pull user from datastore
+				Key key = datastore.newKeyFactory().setKind("User").newKey(appUser);
+				Entity user = datastore.get(key);
+
+				// Get display name, email and lastActive
+				String name = user.getString("name");
+				users.add(new FormattedUser(appUser, name, true, false));
+			}
```

Comment:
style suggestion: to reduce the nesting, I'd prefer continuing if the condition isn't met

```
for (String appUser: nonCollaborators) {
  if (appUser.equals(username)) {
    continue;
  }
  // Otherwise, do everything else here
}
```